Task_HelloWorld/index.html

```diff
@@ -1,15 +1,18 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Hello World Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding below FOR HELLO WORLD TASK;  DO NOT TOUCH the code above -->
-  </body>
-</html>
```

Comment:
The '<br/>' tag should be closed with a forward slash like '<br />' instead of '<br/>'.